src/apps/elgg/mod.rs

```diff
@@ -0,0 +1,50 @@
+use super::{App, State, Step};
+use futures::FutureExt;
+use thirtyfour::prelude::*;
+
+pub const APP: App = App {
+    test: &[
+        Step {
+            name: "login",
+            f: |st: &State| {
+                async {
+                    st.wait(By::XPath("//a[contains(., 'Log in')]"))
```

Comment:
shouldn't this just be `By::LinkText`?

Reply:
I don't remember my exact rational for doing it this way, but since I did a `contains(., 'Log in')` as apposed to a `.='Log in'` or a `text()='Log in'` I suspect the match is not exact.

I'll check when I get a moment. And I'll also check whether or not LinkText concatenates child node text.